c/c++.cpp

```diff
@@ -1,9 +1,9 @@
-#include <iostream> // include API
+#include <iostream> // Include the I/O library
 
 using namespace std;
 
-int main() // the main code portion of a C++ program
+int main(void) // The main function of a C++ program
 {
-   cout << "Hello World" << endl;  //print Hello World on the screen 
-   return 0; // conventional
+   cout << "Hello World" << endl;  // Print Hello World on the screen 
```

Comment:
```suggestion
   cout << "Hello World\n";  // Print Hello World on the screen 
```
[`endl` causes a flush of the output buffer](https://isocpp.org/wiki/faq/input-output#endl-vs-slash-n) in addition to the newline character. This is unnecessary, as the C streams are flushed at the end of the `main()` function anyway -- and, because the C++ streams are synced to their C counterparts by default, this means the text inserted into `cout` so far are flushed as well.

References:
 - [cppreference.com Main function / Explanation](https://en.cppreference.com/w/cpp/language/main_function):
>5) Execution of the return (or the implicit return upon reaching the end of main) is equivalent to first leaving the function normally (which destroys the objects with automatic storage duration) and then calling `std::exit` with the same argument as the argument of the `return`.
 - [cppreference.com `std::exit`](https://en.cppreference.com/w/cpp/utility/program/exit):
>2) all C streams are flushed and closed
 - [cppreference.com `std::ios_base::sync_with_stdio`](https://en.cppreference.com/w/cpp/io/ios_base/sync_with_stdio):
>By default, all eight standard C++ streams are synchronized with their respective C streams.